editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Publish/CheckForChangesButton.tsx

```diff
@@ -50,8 +50,6 @@ export const CheckForChangesToPublishButton: React.FC<{
   const [lastPublishedTitle, setLastPublishedTitle] = useState<string>(
     "This flow is not published yet",
   );
-  const [templateLastPublishedTitle, setTemplateLastPublishedTitle] =
-    useState<string>();
```

Comment:
The source template may have been published _many_ times - this isn't actually a 1:1 relationship so we can remove this in favor of comments only!